src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -458,4 +459,15 @@ public static String convertToRequiredFormat(String date, String hour, String mi
 
     }
 
+    public static String getCustomIdOfTimeZone(double sessionTimeZone) {
+        String sign = sessionTimeZone > 0 ? "+" : "";
+        int hours = (int) sessionTimeZone;
+        int minutes = (int) ((sessionTimeZone - Math.floor(sessionTimeZone)) * 60);
```

Comment:
The formula is not correct for negative time zones which are not `-*` or `-*.5`. 
For example, `-4.25` should return `"GMT-04:15"` but `"GMT-04:45"` instead. 
While there are no such time zones in the list of time zones supported, it is preferred if the logic of this method works for all time zones, not just the ones supported.